resources/lang/en.json

```diff
@@ -363,13 +366,16 @@
   "Group Permissions Updated Successfully": "Group Permissions Updated Successfully",
   "Group Permissions": "Group Permissions",
   "Group": "Group",
+  "group": "group",
```

Comment:
Same thing here regarding capitalization. I think capitalization should be handled with CSS and whether certain words are capitalized or not can be controlled with the bootstrap text helper classes.

Reply:
When searching, I have trouble finding where `Group` and `group` occur. Some are headers and some are select options. Switching all instances of `Group` might be tricky for me. I'm not sure when/where the `group` translation was introduced.

Reply:
This is fine for now. We'll circle back if becomes an issue.